allows ordinary users to specify a remote server, and only throws an error when it doesnt match the local server - inconsistent with many other irc commands (names info motd etc.)

diff -r f9d3f3500a68 ircd/m_admin.c
--- a/ircd/m_admin.c	Sun Jan 11 22:38:40 2009 +0000
+++ b/ircd/m_admin.c	Sun Jan 11 22:38:40 2009 +0000
@@ -120,7 +120,7 @@
   assert(0 != cptr);
   assert(cptr == sptr);
 
-  if (parc > 1  && match(parv[1], cli_name(&me)))
+  if (parc > 1)
     return send_reply(sptr, ERR_NOPRIVILEGES);
 
   return send_admin_info(sptr);
